manager/catalog_manager.go

```diff
@@ -66,13 +64,15 @@ func SetEnv() {
 //Init clones or pulls the catalog, starts background refresh thread
 func Init() {
 	_, err := os.Stat(catalogRoot)
-	if err == nil {
+	if os.IsExist(err) || err == nil {
```

Comment:
If os.stat returns path error, there would be no need to call the code block below. As the path error would indicate that the directory you are looking for, doesn't exist. Unless I'm missing something